### ISSUE TITLE
SSR

### ISSUE THREAD

**Issue by AustinGil** [frameable/el#6]
This looks awesome. Is there already (or planned) a story around SSR support?

**Comment by TobiasNickel** [comment]
how would that look like? having webcomponents be removed to turn into the inner html as string? but somehow still be there to get hydrated? I think you could make a method that executes the render methods in node.js to produce html without webComponent tags. not sure how this would be done when it is nested. 

I think a lib like this is only useful in the client. I think it is a great match with technologies such as ´htmx´.
What are your thoughts?

**Comment by AustinGil** [comment]
Yeah, that's a tricky one. I think most SSR solutions for other frameworks result in a meta framework like Next, Nuxt, etc. That may not be the ideal direction. Might be worth looking at https://enhance.dev/
